pkg/daemon/ceph/osd/daemon.go

```diff
@@ -49,14 +49,25 @@ func StartOSD(context *clusterd.Context, osdType, osdID, osdUUID string, pvcBack
 		logger.Errorf("failed to create config dir %s. %+v", configDir, err)
 	}
 
+	var volumeGroupName string
 	if pvcBackedOSD {
+		logger.Info("PVC backed 2!!")
+		volumeGroupName, err = oposd.GetVolumeGroupName(lvPath)
+		if err != nil {
+			return fmt.Errorf("error fetching volume group name for OSD %s. %+v", osdID, err)
+		}
+
 		if err := updateLVMConfig(context); err != nil {
```

Comment:
Is the LVM config (on the host?) updated just because we manipulated some files? Don't you need to reload LVM or something? Perhaps use '--config' instead for the command?

Reply:
-  lvm config is update on the osd-prepare and osd-daemon pod. Its not updated on the host. 
-  lvm changes that we did to make it work were mostly inspired by this [gist](https://gist.github.com/JohnStrunk/72953876611edea05be7fd302ea6e8ea)
- Need to check and get back to you regarding the latter part of the comment about reloading lv m and using `--config`